### PR TITLE
linter

### PR DESCRIPTION
**Pull request by sonnyp**
merge #13 before

### CODE REVIEW

**Comment by sonnyp** [comment]
I fixed most of the lint issues except this one:

screenshot from 2018-04-17 00-07-37

I'm not sure what to do here, I cannot use eslint-disable because it breaks the bash script.

**Comment by ewnd9** [comment]
Add the file to `.eslintignore` and lint it separately from stdin? (`$ eslint . && tail -n +3 cg.js | eslint --stdin`) :smiley:

**Comment by WebReflection** [comment]
There is no way that line can go or change so I agree with @ewnd9 , just ignore the main entry point since it's also something that is not going to change frequently.

File: packages/assert/test/index.js [hunk 1]
```diff
@@ -16,4 +16,12 @@ const obj3 = {
   }
 };
 
-assert.deepEqual(obj1, obj3);
\ No newline at end of file
+assert.deepEqual(obj1, obj3);
+
+let error
```
Comment (WebReflection) [inline packages/assert/test/index.js]: please respect the style of the project which is with **semi-colons**, also enforce the rule in eslint, thanks.

File: packages/os/src/createSubnet.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  const empty = '0'.repeat(size);
+  return  mask => {
+    const str = ('1'.repeat(parseInt(mask, 10)) + empty).slice(0, size);
+    const out = [];
+    for (let i = 0; i < size; i += segment) {
+      out.push(parseInt(str.substr(i, segment), 2).toString(base));
+    }
+    return out.join(sep);
+  };
+}
```
Comment (WebReflection) [inline packages/os/src/createSubnet.js]: function expression => semi colon

File: packages/os/src/darwin.js [hunk 3]
```diff
@@ -1,9 +1,12 @@
+const createSubnet = require('./createSubnet')
```
Comment (WebReflection) [inline packages/os/src/darwin.js]: semi colon

File: packages/os/src/darwin.js [hunk 4]
```diff
@@ -119,4 +122,4 @@ function parseInterfaces(info) {
     }
   }
   this[info.slice(0, info.indexOf(':'))] = iface;
-}
\ No newline at end of file
+}
```
Comment (WebReflection) [inline packages/os/src/darwin.js]: semi colon

File: packages/os/src/linux.js [hunk 5]
```diff
@@ -1,3 +1,4 @@
+const createSubnet = require('./createSubnet')
```
Comment (WebReflection) [inline packages/os/src/linux.js]: semi colon

File: packages/os/src/darwin.js (hunk 4 above)
Comment (sonnyp) [reply packages/os/src/darwin.js]: Are you sure? there is no semi after function declaration anywhere else.

Comment (WebReflection) [reply packages/os/src/darwin.js]: function declaration !== function expression and this is also my code so yeah, I'm pretty sure.

Comment (sonnyp) [reply packages/os/src/darwin.js]: I meant expression, ok.

**Comment by sonnyp** [comment]
@WebReflection done

**Comment by sonnyp** [comment]
merge ?

**Pull request merged**